model/tensorflow_hub/examples/tfhub_text_classifier/test_textclassifier.py

```diff
@@ -0,0 +1,62 @@
+import os
+import ast
+import sys
+import json
+import tempfile
+import contextlib
+import subprocess
+import unittest.mock
+
+from dffml.model.model import ModelContext, Model, ModelNotTrained
+from dffml.util.os import chdir
+
+
+def sh_filepath(filename):
+    return os.path.join(os.path.dirname(__file__), filename)
+
+
+@contextlib.contextmanager
+def directory_with_csv_files():
+    with tempfile.TemporaryDirectory() as tempdir:
+        with chdir(tempdir):
+            subprocess.check_output(["bash", sh_filepath("train_data.sh")])
+            subprocess.check_output(["bash", sh_filepath("test_data.sh")])
+            yield tempdir
+
+
+class TestExample(unittest.TestCase):
+    def python_test(self, filename):
+        # Path to target file
+        filepath = os.path.join(os.path.dirname(__file__), filename)
+        # Capture output
+        stdout = subprocess.check_output([sys.executable, filepath])
+        lines = stdout.decode().split("\n")
+        # Check the Accuracy
+        self.assertEqual(lines[0], "Accuracy: 1.0")
```

Comment:
I don't think this line will pass with the way the Python script is written at the moment

Reply:
Yes, it doesn't

Reply:
The following output pops at running train which is bugging the tests. I'll look for a way around it in the tests.

`DEBUG:dffml.TextClassifierContext:Training on features: ['sentence']
INFO:dffml.TextClassifierContext:------ Record Data ------
INFO:dffml.TextClassifierContext:x_cols:    6
INFO:dffml.TextClassifierContext:y_cols:    6
INFO:dffml.TextClassifierContext:-----------------------
Model: "model"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
input_text (InputLayer)      [(None,)]                 0         
_________________________________________________________________
keras_layer (KerasLayer)     (None, 20)                389380    
_________________________________________________________________
dense (Dense)                (None, 256)               5376      
_________________________________________________________________
dense_1 (Dense)              (None, 3)                 771       
=================================================================
Total params: 395,527
Trainable params: 395,527
Non-trainable params: 0
_________________________________________________________________
Train on 6 samples
Epoch 1/30
6/6 [==============================] - 0s 81ms/sample - loss: 3.7750e-07 - accuracy: 1.0000
`

Reply:
Hmmm, weird, ya probably best to just skip to parsing the last lines in stdout